src/main/java/com/example/music/backend/user/service/UserServiceV1.java

```diff
@@ -0,0 +1,66 @@
+package com.example.music.backend.user.service;
+
+import com.example.music.backend.common.exception.NotFoundException;
+import com.example.music.backend.verification.domain.VerificationToken;
+import com.example.music.backend.verification.repository.VerificationTokenRepository;
+import com.example.music.backend.user.converter.UserDtoConverter;
+import com.example.music.backend.user.domain.User;
+import com.example.music.backend.user.dto.UserDto;
+import com.example.music.backend.user.exception.UserAlreadyExistException;
+import com.example.music.backend.user.repository.UserRepository;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceV1 implements UserService {
```

Comment:
Rename it, please. Something like UserServiceImpl looks better

Reply:
renamed